src/compile_model_manifest.py

```diff
@@ -0,0 +1,30 @@
+import os
+import requests
+import yaml
+import re
+import json
+
+compiled_models = []
+preserved_keys = ["config_url", "applications", "download_url", "name", "description", "cite", "authors", "documentation", "tags", "covers"]
+assert 'url' not in preserved_keys
+with open('./src/models.yaml') as f:
+    model_list = yaml.safe_load(f.read())
+    for item in model_list:
+        config_url = item['config_url']
+        response = requests.get(config_url)
+        if response.status_code == 200:
+            model_config = yaml.safe_load(response.content)
+
+            # merge item from models.yaml to model config
+            model_config.update(item)
+            model_info = {}
+            for k in model_config:
+                if k in preserved_keys:
+                    model_info[k] = model_config[k]
+            compiled_models.append(model_info)
+
+with open('src/manifest_template.json', 'rb') as t:
```

Comment:
Using `pathlib.Path` reduces indentation level and gives nice defaults.
```suggestion
template_to_read = Path('src/manifest_template.json')
template = json.loads(template_to_read.read_text())
template['models'] = compiled_models
template_to_write = Path('manifest.model.json')
template_to_write.write_text(json.dumps(template, sort_keys=True, indent=2))
```

We can make it binary files if you need to, but I don't see why we need that.

I removed `separators` as the default is the same from Python 3.4, which is already end of life.